python-mini-projects/projects/capture_screenshot/screenshot.py

```diff
@@ -0,0 +1,47 @@
+import os
+import argparse
+import pyautogui
+import time
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("-p", "--path", help="absolute path to store screenshot.", default=r"./images")
+parser.add_argument("-t", "--type", help="h (in hour) or m (in minutes) or s (in seconds)", default='h')
+parser.add_argument("-f", "--frequency", help="frequency for taking screenshot per h/m/s.", default=1, type=int)
+
+args = parser.parse_args()
+
+
+sec = 0.
+
+if args.type == 'h':
+    sec = 60 * 60 / args.frequency
+elif args.type == 'm':
+    sec = 60 / args.frequency
+
+if sec < 1.:
+    sec = 1.
+    
+
+if os.path.isdir(args.path) != True:
+    os.mkdir(args.path)
+
+
+try:
+    while True:
+        t = time.localtime()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Use the `datetime` module for time zone conversions. The older `time` module doesn&#39;t work consistently with multiple time zones and shouldn&#39;t be used.

[Learn more](https://docs.python.org/3/library/time.html#module-time.)